plugins/out_stackdriver/stackdriver.c

```diff
@@ -2495,23 +2495,29 @@ static int stackdriver_format_test(struct flb_config *config,
 static void add_record_metrics(struct flb_stackdriver* ctx,
                                uint64_t ts,
                                int val,
-                               int response_code) {
-    char response_code_label[32];
-
-    /* convert status to string format */
-    snprintf(response_code_label, sizeof(response_code_label) - 1, "%i",
-             response_code);
-    char* name = (char*) flb_output_name(ctx->ins);
-
-    /* processed records total */
-    cmt_counter_add(ctx->cmt_proc_records_total, ts, val, 2,
-                    (char* []) {response_code_label, name});
+                               int response_code,
+                               int grpc_code)
+{
+  char grpc_code_label[32];
+  char response_code_label[32];
+  char* name;
```

Comment:
[Optional] You should be able to initialize this here (as long as it's at the start of the function)…